vxhunter/vxworks_symtab.py

```diff
@@ -57,5 +55,18 @@ def define_symbol_table(vx_ver):
         exit()
 
     # Try to get the symbol table/define the symbols.
-    define_symbol_table(vx_ver)
-
+    try:
+       define_symbol_table(vx_ver)
+    except (ghidra.util.exception.InvalidInputException, RuntimeError):
+        # stuff went wrong!!! we should try see if rebasing will work
+        if 0 == get_image_base().offset and 1 == len(get_memory_blocks()):
+            # 0 should not be a valid memory address, and we have a monolithic image
```

Comment:
In ARM images, the EVT is often at address 0. I think you can probably just ignore the base of 0 and assume that if something went wrong, the base is bad

Reply:
That's fair, but if running manually and i've set a base manually, then i don't want the script to change that on me... I guess it's a preference